server/plugin/plugin_test.go

```diff
@@ -88,8 +87,6 @@ func TestPluginOnActivate(t *testing.T) {
 		},
 		"same version as minimumServerVersion": {
```

Comment:
This test case name should be changed.

```suggestion
		"all fine": {
```

Reply:
Done